bots/coniksbot/coniksbot.go

```diff
@@ -1,3 +1,5 @@
+// Package coniksbot provides the main executable for
```

Comment:
I wasn't sure which one to choose. godoc.org shows it as `bots/coniksbot` so that's why I went for that. Which will you use for the keyserver? As long as we're consistent, I'm fine with either one.